client/wfm/README.md

```diff
@@ -29,19 +23,3 @@ In order to use this module, begin by installing all dependencies:
 If inside the [lerna](http://lernajs.io)-based mono repo: `lerna bootstrap --scope=@raincatcher/wfm --include-filtered-dependencies`
 
 For standalone usage: `npm install`
-
-For implementing a base flow, see the [examples folder](./examples/).
```

Comment:
We missing example code (I know it may be strange to do this but as we agreed on we need to have examples and try to document some specifics like new step creation etc.)